src/translation/translation_ja_JP.ts

```diff
@@ -0,0 +1,4341 @@
+        <source>Enables support for very small network audio packets. These network packets are only actually used if the sound card buffer delay is smaller than %1 samples. The smaller the network buffers, the lower the audio latency. But at the same time the network load and the probability of audio dropouts or sound artifacts increases.</source>
+        <translation>非常に小さいネットワークオーディオパケットのサポートを有効にします。これらのネットワークパケットは、サウンドカードバッファ遅延が%1サンプルより小さい場合にのみ実際に使用されます。ネットワークバッファが小さいほど、オーディオレイテンシが低くなります。しかし同時に、ネットワーク負荷とオーディオドロップアウトまたはサウンドアーティファクトの可能性が増加します。</translation>
+    </message>
+    <message>
+        <source>Some sound card drivers do not allow the buffer delay to be changed from within %1. In this case the buffer delay setting is disabled and has to be changed using the sound card driver. On Windows, use the ASIO Device Settings button to open the driver settings panel. On Linux, use the JACK configuration tool to change the buffer size.</source>
+        <translation type="vanished">一部のサウンドカードドライバーは、%1内からバッファ遅延を変更することを許可しません。この場合、バッファ遅延設定は無効になり、サウンドカードドライバーを使用して変更する必要があります。Windowsでは、ASIOデバイス設定ボタンを使用してドライバー設定パネルを開きます。Linuxでは、JACK設定ツールを使用してバッファサイズを変更します。</translation>
+    </message>
+    <message>
+        <source>If no buffer size is selected and all settings are disabled, this means an unsupported buffer size is in use by the driver. %1 will still work with this setting but may have restricted performance.</source>
+        <translation type="vanished">バッファサイズが選択されておらず、すべての設定が無効になっている場合、ドライバーでサポートされていないバッファサイズが使用されていることを意味します。%1はこの設定でも動作しますが、パフォーマンスが制限される可能性があります。</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlg.cpp" line="59"/>
+        <source>Controls the relative levels of the left and right local audio channels. For a mono signal it acts as a pan between the two channels. For example, if a microphone is connected to the right input channel and an instrument is connected to the left input channel which is much louder than the microphone, move the audio fader in a direction where the label above the fader shows %1, where %2 is the current attenuation indicator.</source>
+        <translation>左右のローカルオーディオチャンネルの相対レベルを制御します。モノラル信号の場合、2つのチャンネル間のパンとして機能します。例えば、マイクが右入力チャンネルに接続されていて、楽器がマイクよりもはるかに大きい左入力チャンネルに接続されている場合、フェーダー上のラベルが%1を示す方向にオーディオフェーダーを動かしてください。%2は現在の減衰インジケーターです。</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlg.cpp" line="122"/>
+        <source>Audio Device</source>
+        <translation>オーディオデバイス</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlg.cpp" line="123"/>
+        <source>Under the Windows operating system the ASIO driver (sound card) can be selected using %1. If the selected ASIO driver is not valid an error message is shown and the previous valid driver is selected. Under macOS the input and output hardware can be selected.</source>
```

Comment:
We should really be consistent with macOS instead of MacOS throughout the code...

But that's not your problem ;-)